PV_Excess_Control/pyscript/pv_excess_control.py

```diff
@@ -269,6 +269,7 @@ def on_time():
             # ----------------------------------- go through each appliance (highest prio to lowest) ---------------------------------------
             # this is for determining which devices can be switched on
             instances = []
+            switched_off_aplliance_to_switch_on_higher_prioritized_one = False
```

Comment:
typo in this var name

```suggestion
            switched_off_appliance_to_switch_on_higher_prioritized_one = False
```